mappings/net/minecraft/client/gui/screen/multiplayer/report/AbuseReportScreen.mapping

```diff
@@ -0,0 +1,24 @@
+CLASS net/minecraft/unmapped/C_mwsdyxmu net/minecraft/client/gui/screen/multiplayer/report/AbuseReportScreen
```

Comment:
This is confusing, because it doesn't extend `AbstractAbuseReportScreen` - I think it needs a better name... (`AbstractAbuseReportScreen` is extended by `ChatAbuseReportScreen`, `SkinAbuseReportScreen`, `UsernameReportScreen` [oops, I'll go fix that one])